utils.js

```diff
@@ -1,3 +1,19 @@
-export function findLanguage(languages, locale) {
-  return languages.find((language) => language.code === locale)
+import { defaultLocale } from './default'
+import { languages } from './constants'
+
+export function findLanguage(locale) {
+  return getExpandedLanguages().find((language) => language.code === locale)
+}
+
+export function getLanguageFromUrl(url) {
+  return url.split('/')[1] || defaultLocale
```

Comment:
```suggestion
  return url.split('/')[1]
```

Reply:
if no locale is specified in the url it should return the default